.github/workflows/cargo-file-checks.yml

```diff
@@ -0,0 +1,24 @@
+name: Cargo file checks
```

Comment:
If you think this is a good idea. After merging this PR I can create a github action to use the same code in other rust repositories.